.travis.yml

```diff
@@ -31,6 +34,9 @@ matrix:
       env: WP_TRAVISCI=phpcs
     - php: 5.3
       env: WP_VERSION=latest
+  allow_failures:
+    - php: 5.6
+      env: WP_TRAVISCI=phpcs
```

Comment:
Allow failures in PHPCS for standard plugins to eliminate churn for common use cases.